app/User.php

```diff
@@ -30,4 +30,8 @@ class User extends Authenticatable
 
     protected $test = 'testtest';
     protected $test2 = 'test2test2';
+
+    private function newPR() {
+        
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$test5" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://relisonar.relipasoft.com:9000/coding_rules#rule_key=php%3AS1068)